README.md

```diff
@@ -36,30 +36,34 @@ The React Native example app is hosted in a yarn-based mono-repo. There are shar
 
 Get the code by cloning this repo using git.
 
-``` bash
+```bash
 git clone git@github.com:dolbyio-samples/comms-app-react-native-videocall.git
-
 ```
 
-Once cloned, open a terminal in the project directory, and install dependencies with yarn.
+Go to the project directory, and install the dependencies with yarn.
 
-``` bash
+```bash
 cd comms-app-react-native-videocall
 yarn install
+```
+
+If you do not have it already, install `ios-deploy` globally on your machine.
 
+```bash
+npm install -g ios-deploy
```

Comment:
Is `iOS-deploy` installation a required step? Where is it used ?

Reply:
It is required when you want to deploy the iOS app on your phone. I've tried to add it as a devDependency but the scripts won't work that way, so you must install it globally.

Reply:
@FabienLavocat What is this iOS app? Is it not the example app in this repo? and developed by you? Is it a one-time-only step?
To build and run the example app, I don't remember any team member or myself needing to do this.